DigitalLearningSolutions.Web.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -1292,6 +1239,22 @@ public void CreateDelegateAccountForExistingUser_adds_new_delegate_to_groups()
             );
 
             // Then
+            A.CallTo(
+                () => groupsService.AddNewlyRegisteredUserToAddNewRegistrantGroups(
+                    NewDelegateIdAndCandidateNumber.Item1,
+                    A<DelegateRegistrationModel>.That.Matches(
+                        m =>
+                            m.Answer1 == model.Answer1 &&
+                            m.Answer2 == model.Answer2 &&
+                            m.Answer3 == model.Answer3 &&
+                            m.Answer4 == model.Answer4 &&
+                            m.Answer5 == model.Answer5 &&
+                            m.Answer6 == model.Answer6 &&
+                            m.JobGroup == userEntity.UserAccount.JobGroupId &&
+                            m.Centre == model.Centre
+                            ))
+            ).MustHaveHappenedOnceExactly();
+            /*
             A.CallTo(

```

Comment:
Remove commented out code

Reply:
What's distressing me here is I checked this file twice.